frontend/__tests__/components/command-line-tools.spec.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+
+import { CommandLineTools } from '../../public/components/command-line-tools';
+
+describe('CommandLineTools', () => {
+  let wrapper;
+  const obj = {
+    data: [
+      {
+        apiVersion: 'console.openshift.io/v1',
+
+        kind: 'ConsoleCLIDownload',
+
+        metadata: {
+          creationTimestamp: '2020-03-16T12:31:49Z',
+          generation: 1,
+
+          name: 'helm-download-links',
+
+          resourceVersion: '10488',
+
+          selfLink: '/apis/console.openshift.io/v1/consoleclidownloads/helm-download-links',
+
+          uid: '4984e824-96b7-465a-a67b-46b407aee972',
+        },
+        spec: {
+          description:
+            'Helm 3 is a package manager for Kubernetes applications which enables defining,↵installing, and upgrading applications packaged as Helm Charts.↵Helm 3 is a [Technology Preview](https://access.redhat.com/support/offerings/techpreview) feature on OpenShift 4.↵',
```

Comment:
Since this is a unit test, you can also eliminate unnecessary fields, condense long fields, etc.
I don't think you need the `uid`, `selfLink`, etc at all.

Reply:
I dropped all the fields not required for the test (removing uid leads to a key warning, so I left that one in).